lib/sync/meson.build

```diff
@@ -0,0 +1,38 @@
+libephysync_sources = [
+  'ephy-sync-crypto.c',
+  'ephy-sync-service.c',
+  'ephy-synchronizable-manager.c',
+  'ephy-synchronizable.c'
+]
+
+libephysync_deps = [
+  glib_dep,
+  gmp_dep,
+  hogweed_dep,
+  json_glib_dep,
+  libsecret_dep,
+  libsoup_dep,
+  nettle_dep,
+  webkit2gtk_dep
+]
+
+libephysync_includes = include_directories(
+  '.',
+  '..',
+  '../history',
+  '../widgets',
+  '../..',
+  '../../embed'
```

Comment:
> The easiest thing would be to add a new GObject interface under lib/sync, say EphyTabList. Perhaps you can come up with a better name than that. Have EphyWindow implement it. You just need one function, really, ephy_tab_list_get_open_tabs.

This solves a part of the dependency, but not all. We still need to get all the tabs from all the windows, right? And to get all the windows I need `gtk_application_get_windows (GTK_APPLICATION (embed_shell))` which still requires embed. Am I missing something? Is there another way to retrieve all `EphyWindow`s?